pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
+
+`composer require {{user/package-name}}`
+
+- Install all dependencies for this project:
+
+`composer install`
+
+- Remove a package from this project:
```

Comment:
That might work, but `composer remove` also deletes the package if it's installed too.